analyzers/packaging/tools-vbnet/install.ps1

```diff
@@ -48,5 +48,8 @@ $project.Object.AnalyzerReferences.Add($analyzerFilePath)
 $analyzerFilePath = Join-Path $analyzersPath "SonarAnalyzer.dll"
 $project.Object.AnalyzerReferences.Add($analyzerFilePath)
 
+$analyzerFilePath = Join-Path $analyzersPath "SonarAnalyzer.CFG.dll"
```

Comment:
NITPICK: WDYT think about making the same order of steps as for the chsarp install script? Theoretically it should ease the update of the scripts if the same action has to be added to both. Same goes for the uninstall script.

Reply:
It's the CSharp that was wrong. Fixed.